src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -28,8 +28,8 @@
 
     private static final Logger log = Logger.getLogger();
 
-    private static final Map<String, String> TIME_ZONE_CITIES_MAP = new HashMap<>();
-    private static final List<Double> TIME_ZONE_VALUES = new ArrayList<>();
+    private static final Map<ZoneId, String> TIME_ZONE_CITIES_MAP = new HashMap<>();
```

Comment:
If we make this a `LinkedHashMap`, we can get rid of `TIME_ZONE_VALUES` entirely - just use `TIME_ZONE_CITIES_MAP.values()` to build the list.